packages/driver/cypress/e2e/issues/3890.cy.js

```diff
@@ -12,40 +12,38 @@ describe('issue 3890 overwriting cy.route command', () => {
     })
   })
 
-  it('stores route as an alias', () => {
-    // sanity test before overwriting cy.route in the next test
+  it('stores intercept as an alias', () => {
```

Comment:
I'd noticed, rewriting the aliasing code, that .route was well tested but not .intercept. Glad to see that everything still passes when we switch tests over to the newer command!